website/.storybook/preview.js

```diff
@@ -0,0 +1,20 @@
+import "!style-loader!css-loader!postcss-loader!tailwindcss/tailwind.css";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
+
+import * as NextImage from "next/image";
+
+const OriginalNextImage = NextImage.default;
+
+Object.defineProperty(NextImage, "default", {
+  configurable: true,
+  value: (props) => <OriginalNextImage {...props} unoptimized />,
+});
```

Comment:
oof...

what's the problem here exactly?

Reply:
little hacky solution because next handles images in a way, that simply adding the public folder to storybook would not work... i used the solution from [here](https://dev.to/jonasmerlin/how-to-use-the-next-js-image-component-in-storybook-1415)

Reply:
Damn, very nasty, could you  please document this in the code? so that future developers can understand why.